hwdb/60-sensor.hwdb

```diff
@@ -68,3 +68,10 @@ sensor:modalias:platform:lis3lv02d:dmi:*svn*Hewlett-Packard*:*pn*HPEliteBook8560
 #########################################
 sensor:modalias:acpi:BMA250*:dmi:*svn*WinBook*:*pn*TW100*
  ACCEL_MOUNT_MATRIX=0, -1, 0; -1, 0, 0; 0, 0, 0
+
+
+#########################################
+# Complex 11t
+#########################################
+sensor:modalias:acpi:*KIOX000A*:*
```

Comment:
You need to add the "dmi" match here. The output of:
```
udevadm info --path=/devices/virtual/dmi/id | grep MODALIAS
```
will give you the full DMI string for the machine/tablet. Select the minimum information that's necessary to uniquely identify this device. Also see the match for the "Winbook TW100" tablet just above as an example.